src/main/java/com/bandwidth/sqs/client/SqsClient.java

```diff
@@ -91,25 +86,19 @@ public SqsClient(SqsRequestSender requestSender, Function<String, T> map, Functi
      * @param queueConfig Configuration of the SQS queue
      * @return an SqsQueue
      */
-    public Single<SqsQueue<T>> upsertQueue(SqsQueueConfig queueConfig) {
+    public Single<SqsQueue<String>> upsertQueue(SqsQueueConfig queueConfig) {
         return upsertQueue(queueConfig, SqsQueueClientConfig.builder().build());
     }
 
-    /**
-     * Returns a String SqsClient builder, but can be converted to any type of
-     * SqsClient by providing your own serializer / deserializer
-     */
-    public static SqsClientBuilder<String> builder() {
-        //default builder has "no-op" serializers
-        return new SqsClientBuilder<>((str) -> str, (str) -> str);
+    public static SqsClientBuilder builder() {
```

Comment:
The "other convention" is for `@Immutable` objects, this was a builder I built myself. Not sure if there is much of a benefit of using an inner class for the builder.